<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowLive</class>
 <widget class="QDialog" name="WindowLive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRSF - Main</string>
  </property>
  <widget class="QPushButton" name="btn_Activate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Stop">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_SignalSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Signal set:</string>
   </property>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Track">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>94</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Track:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Track">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_SignalSetName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cb_RecordTelemetry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Telemetry</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Status">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_BMSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Brake marker set:</string>
   </property>
   <widget class="QComboBox" name="cb_BrakeMarkerSets">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_Track_bm">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Track_bm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Track:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
